config/application.rb

```diff
@@ -20,7 +20,7 @@ class Application < Rails::Application
       'OFFENDER_MANAGEMENT_ALLOCATION_API',
       'http://localhost:8000'
     )
-    config.sentry_dsn = ENV.fetch('SENTRY_DSN', nil)
+    config.sentry_dsn = ENV.fetch('SENTRY_DSN', 'nil')
```

Comment:
I'm a bit unsure about the `'nil'`, is this because something somewhere else requires it to be a string?

Reply:
mmm ... does not seem right. Will check @rossjones

Reply:
I've removed the default value